setup.py

```diff
@@ -1,49 +1,53 @@
-from setuptools import setup
+from setuptools import setup, find_packages
 import re
-"""
-Python library that allows you to interact programmatically with an instance of
-the Genestack platform.
-"""
 
 regex = re.compile(r'^--.*$')
 # Read requirements file to download deps with filtering params.
 with open('requirements.txt') as f:
     required = [line for line in f.read().splitlines() if not regex.match(line)]
 
 # Import version directly, without execute __init__.py script
-exec(open('genestack_client/version.py').read())
+exec(open('odm_sdk/version.py').read())
 
 setup(
-    name='genestack_client',
+    name='odm-sdk',
     install_requires=required,
     version=__version__,
-    packages=['genestack_client', 'genestack_client.settings', 'genestack_client.scripts'],
+    packages=find_packages(),
     url='https://github.com/genestack/python-client',
     license='MIT',
     author='Genestack Limited',
     author_email='',
-    description='Genestack Python Client Library',
+    description='SDK for interacting with the Open Data Manager',
     long_description=__doc__,
     long_description_content_type="text/markdown",
-    python_requires='>=3.7',
+    keywords=['genestack', 'odm', 'import', 'share', 'create',
+              'delete', 'curate', 'genomics', 'api'],
+    include_package_data=True,
+    python_requires='>=3.10',
     entry_points={
         'console_scripts': [
```

Comment:
wondering how far we are from creating one entry point with options inside?..

Reply:
Honestly, IDK.

For me it's pretty far away taking into account that we should support windows installations for our CLI. At the moment python and pip do it for us.